docs/docs/dart_api/manager.md

```diff
@@ -118,6 +118,23 @@ Any rows that meet the specified condition will be deleted.
 
 {{ load_snippet('manager_delete','lib/snippets/dart_api/manager.dart.excerpt.json') }}
 
+
+## Annotations
+
+It's quite common that you want to perform some complex calculations you data. Drift provides an easy way to write SQL expressions which can be read and/or used in filters and orderings.
```

Comment:
This sentence is broken, and I think this feature generally deserves a better introduction. Maybe the story should be that manager queries are great when selecting whole rows + references, but sometimes its beneficial to push more logic into SQL e.g. efficient aggregates? Then we can link + mention drift's existing support for these expressions and finally present this as the way to use them with manager APIs?

Reply:
I lan on rewriting most of the docs and giving annotations its own page, so i didn't put much thought into these temporary docs.

But i completely agree, I'll update it